designer/pydesigner.py

```diff
@@ -200,6 +200,14 @@
                     help='Print out all output. This is a very messy '
                     'option. We recommend piping output to a text file '
                     'if you use this option.')
+parser.add_argument('--adv', action='store_true',
+                    help='Disables safety checks for advanced users who '
+                         'want to force a preprocessing step. WARNING: '
+                         'THIS FLAG IS FOR ADVANCED USERS ONLY WHO FULLY '
+                         'UNDERSTAND THE MRI SYSTEM AND ITS OUTPUTS. '
+                         'RUNNING WITH THIS FLAG COULD POTENTIALLY '
+                         'RESULT IN IMPRECISE AND INACCURATE RESULTS, '
+                         'OR A THERMONUCLEAR WAR.')
```

Comment:
Okay, that's excessive, let's get rid of that.

Reply:
Toned down in cf46e11ee05fd0351939df6ab17680c6357e86b6